app/app.model.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection");
+
+exports.selectAllTopics = (req) => {
+  if (Object.keys(req.query).length !== 0 || Object.keys(req.params).length !== 0) {
```

Comment:
I can see what you are thinking here.  Topics endpoint doesn't necessarily require a query (definitely doesn't require params unless you want to fetch topic by id or something, which would be a separate endpoint!).  But I can see you've tested for it to a degree, and it would be technically possible. But you have not tested for the behaviour of a valid query.  If you want this feature then fine, build it out fully.  But it's not fully required, so don't spend time on it now!